### DISCUSSION TITLE
Consider implementing supabase / stripe-sync-engine with RPC inside directly in Postgres DB

### DISCUSSION THREAD

**Discussion by kav** [supabase/supabase#6442] [Feature Requests]
Hi y'all, we've got stripe webhooks hitting a postgres function directly that updates the stripe tables and it works great. I just ran across the supabase stripe sync engine and was curious have you looked at doing it inside PG? I'd be happy to help or if you considered and dismissed the approach I'd love to know why so I don't run right into something similar with my approach

**Comment by aplyd** [comment]
@kav did you ever implement this? if so, how'd it go?

**Reply by kav** [reply]
I did, it works great. I also briefly went on a tangent with the Stripe Postgres FDW https://github.com/supabase/wrappers which isn't quite where I needed. The half table half api feel of the FDW didn't work nearly as well as the webhook to function approach
